cron/k8s/controller.yaml

```diff
@@ -52,7 +52,11 @@ spec:
           containers:
             - name: controller
               image: gcr.io/openssf/scorecard-batch-controller:stable
-              args: ["--config=/etc/scorecard/config.yaml", "cron/internal/data/projects.csv"]
+              args: [
+                "--config=/etc/scorecard/config.yaml", 
+                "cron/internal/data/projects.csv", 
+                "cron/internal/data/gitlab-projects-selected.csv"
```

Comment:
what are your thoughts on doing this in the release controller config instead?

Reply:
What is the difference between this controller and the release controller?

Reply:
The controller enumerates the weekly 1.2M repos for the production workers.
The release controller is the nightly test that does 5000 repos for the staging worker that moves commits from `latest` to `stable`.

My opinion is that the experimental features should stay in the release/nightly worker to not affect the cron.